app/build.gradle.kts

```diff
@@ -87,12 +88,12 @@ dependencies {
     ksp(libs.room.compiler)
     implementation(project(":fakegen"))
     ksp(project(":fakegen"))
+    implementation(libs.kotlinx.coroutines.core)
 
-    testImplementation(libs.junit)
-    androidTestImplementation(libs.androidx.test.ext.junit)
-    androidTestImplementation(libs.espresso.core)
-    androidTestImplementation(platform(libs.compose.bom))
-    androidTestImplementation(libs.ui.test.junit4)
```

Comment:
Removed Junit4 libraries + UI testing libraries for now. I guess we should decided if we want to do UI tests.

Reply:
To be totally honest, I am not aware of the newer features of JUnit 5 so I'm going to blindly approve for the time being

Reply:
There are a few new features, but for the most part it's the same as JUnit 4